v3/pkg/shell/shell.go

```diff
@@ -489,6 +493,287 @@ func copyResponse(rw http.ResponseWriter, resp *http.Response) error {
 	return err
 }
 
+type VirtualMachineInputTask struct {
+	VMId              string             `json:"vm_id"`
+	VMName            string             `json:"vm_name"`
+	TaskInputCommands []hfv1.Task        `json:"task_command"`
+}
+
+type VirtualMachineOutputTask struct {
+	VMId               string              `json:"vm_id"`
+	VMName             string              `json:"vm_name"`
+	TaskOutputCommands []TaskOutputCommand `json:"task_commands_output"`
```

Comment:
Okey sorry that i bring this up now but maybe do it this way:


VMid, VMname, hfv1.Task and then TaskOutput

TaskOutput has ActualOutputValie, ActualReturnCode, Success